supervisor/v1/supervisor.proto

```diff
@@ -102,6 +102,9 @@ message PresentationConfig {
     // specified here. Once we've started using encryption on a volume it's
     // impossible to undo it. The opposite is also true.
     optional common.v1.VolumeCrypto opt_volume_crypto = 8;
+
+    // if set, we're presenting a snapshot
```

Comment:
Try and follow the style of the file. All of the other comments start with a capital letter